### DISCUSSION TITLE
Your Guide to Universe 2024 Speaker Posts

### DISCUSSION THREAD

**Discussion by novium2** [community/community#141919] [Other Feature Feedback, Questions, & Ideas]
## Countdown to Universe 2024 is on!

Universe 2024 is fast approaching, and to celebrate we're going to spend the next two weeks bringing everything Universe to Community Discussions.

To help us get started, tell us how you plan to attend Universe by responding below with:

🚀 if you're planning on attending in person  
🎉 if you're planning to attend virtually.

### Professional Development:
- #141485
- #141819

### Security:
-  #141479
- #141572 
- #139507 
- #141704

### Copilot & AI
- #142312
- #141293
- #142105
- #142971
- #142536

### Enterprise
- #138285
- #141479
- #141502
- #142848
- #143136

If you'd like to see more of the speakers, sessions, and talks that will be taking place at Universe 2024, [click here](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog?tab.day=20241029).

**Comment by VishalKotekar** [comment]
Universe 2024 is approaching, and GitHub invites users to participate in a two-week celebration in Community Discussions. You can share whether you plan to attend in person or virtually. There are various speaker sessions on topics like Copilot, GitHub and Azure integration, open-source contributions, and more. For further details on the event and sessions, you can visit the provided link.

More details on Universe 2024 are available [here](https://github.com/orgs/community/discussions/141919).

**Comment by FreeTellurian** [comment]
жду, готовьтесь удивить нас.
;)

**Reply by ettaboyle** [reply]
Hi @FreeTellurian ✨ , thanks for participating and we're glad you're looking forward to Universe 2024!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. 

**Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.** Please post in English going forward. Thank you!

**Comment by Abraham-Flutterwave** [comment]
i'll be attending virtually.

**Comment by fbrandao2k** [comment]
I'm going to participate virtually as well. It will start soon!

**Comment by nitendra2211** [comment]
I'm going to participate virtually as well. It will start soon!

**Comment by Anudhyan** [comment]
Will there be swags?

**Comment by JuanTorchia** [comment]
🎉 I’ll be tuning into Universe 2024 virtually again—third year running. No airports, no crowds, just me, a screen, and whatever chaos unfolds. It's not the same as being there, but it gets the job done. Let’s see what GitHub has up its sleeve this time.

**Comment by samus-aran** [comment]
Wooo! Live stream is happening in 7 minutes 😃

**Comment by aspratle** [comment]
Excited to be here virtually!!!

**Comment by Nathan-Bransby-NMT** [comment]
I'm so pumped for all the new Copilot features! :octocat:

**Comment by trxdave** [comment]
Following Universe 2024 online! Excited to see the latest GitHub innovations and insights 🚀

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by Vijayprabhudasingale1987** [comment]
![IMG-20241009-WA0000](https://github.com/user-attachments/assets/171a9413-fb0c-473a-b114-1fb6b7f5623a)
Vijay Prabhudas ingale

**Comment by Vijayprabhudasingale1987** [comment]
![Heather-Monaghan_avatar_1659723319-300x300](https://github.com/user-attachments/assets/a30452f8-f006-489a-9458-6537dc07dc1a)
![Solvay_conference_1927](https://github.com/user-attachments/assets/79249901-cf3a-432c-b909-9021bbc35a41)
![The_Telephone_Service_Carries_On,_London,_January_1942_D6437](https://github.com/user-attachments/assets/9b508b19-d17f-4e6d-93df-e7a9cc50ce26)
![Uploading Frontansicht_des_Hauptgebäudes_der_Humboldt-Universität_in_Berlin.jpg…]()

**Comment by Vijayprabhudasingale1987** [comment]
![iss-blowout-updated-view-web-version-010323-1](https://github.com/user-attachments/assets/648a3a93-cf99-4fda-b474-68dc88e0e906)
[Uploading dss43-6_fs.webp…]()

**Comment by Vijayprabhudasingale1987** [comment]
![Uploading 9352713724_c2edddf21e_z.jpg…]()

**Comment by a1baradi** [comment]
Oopps
